packages/js/src/ui/components/Notification/NotificationList.tsx

```diff
@@ -0,0 +1,111 @@
+import { For, ParentProps, Show } from 'solid-js';
+import { FetchFeedArgs } from '../../../feeds';
+import { useFeedInfiniteScroll } from '../../api';
+import { useLocalization } from '../../context';
+import { useStyle } from '../../helpers';
+import { EmptyIcon } from '../../icons/EmptyIcon';
+import { SkeletonAvatar, SkeletonText } from '../primitives/Skeleton';
+
+const NotificationListContainer = (props: ParentProps) => {
+  const style = useStyle();
+
+  return (
+    <div class={style('notificationListContainer', 'nt-w-[25rem] nt-h-[37.5rem] nt-overflow-auto')}>
```

Comment:
```suggestion
    <div class={style('notificationListContainer', 'nt-w-[25rem] nt-h-full nt-overflow-auto')}>
```

`37.5rem` is too much, but because it's in the flex container the height is restricted to available space

Reply:
and the `width` should not be set here but defined by the list item content, otherwise, if someone creates the custom list item they will be restricted by the width of the container and would have to override it